addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -66,7 +66,12 @@ options.registry.SnippetPopup = options.Class.extend({
     /**
      * @override
      */
-    onTargetHide: async function () {
+    onTargetHide: async function (willSave) {
+        if (willSave) {
+            // Do not update option while saving.
+            this.$target.off('hide.bs.modal.SnippetPopup');
+            this._removeIframeSrc();
+        }
```

Comment:
That comment is not enough, I don't understand what this actually does. Why is it necessary to prevent the fact `this.trigger_up('snippet_option_visibility_update', {show: false});` is called? I understand why we would need it as perf improvement but why is it needed in stable? And if it is needed in stable, should not the fix be global? Something like "Before calling `toggleTargetVisibility` when saving, stop listening to `snippet_option_visibility_update` events" ?